combinator challenge/flexbox-challenge/style.css

```diff
@@ -0,0 +1,47 @@
+section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+
+.container {
+  display: flex;
+  height: 300px;
+  width: 900px;
+  border: 2px solid black;
+  align-items: flex-start;
+  justify-content: center;
+}
+
+/* .cards {
+  display: flex;
+  
+  justify-content: center;
+  align-items: center;
+
+} */
+
```

Comment:
You can remove these commented out styles.

Reply:
Thanks will do!